### ISSUE TITLE
Scheduler worker constantly dies

### ISSUE THREAD

**Issue by NailBash** [MISP/MISP#6210]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | CentOS 7
| PHP version               | 7.2
| MISP version / git hash   | 2.4.19, 5fc0fb0f687c6409e16a320a05f4d471c552905

### Expected behavior
Scheduler worker works fine

### Actual behavior
Scheduler worker constantly dies

### Logs, screenshots, configuration dump, ...
![изображение](https://user-images.githubusercontent.com/29750691/90603707-1c95ee80-e215-11ea-880d-bd140142f3ba.png)

![изображение](https://user-images.githubusercontent.com/29750691/90603770-37686300-e215-11ea-947a-913b7e2c5858.png)

Also, after restart of workers they do not restart automatically so I pushing all buttons on workers page, to launch them. All workers starts to run excluding scheduler.

**Comment by iglocska** [comment]
That looks like a permission issue. This can be caused by the worker being started by the wrong user and the log file being created for said user. Make sure that: 

1. create files in /var/www/MISP/app/tmp/logs/ is writable by the apache user
2. all files in /var/www/MISP/app/tmp/logs/* are owned by the apache user.

**Comment by NailBash** [comment]
![изображение](https://user-images.githubusercontent.com/29750691/90630909-4cf08380-e23b-11ea-9221-3815770beead.png)

![изображение](https://user-images.githubusercontent.com/29750691/90630962-5f6abd00-e23b-11ea-80f0-aeac5bc7d3ef.png)

Everything ok I think, hmm

**Comment by iglocska** [comment]
The culprit:

![image](https://user-images.githubusercontent.com/3668672/90631684-73a2c000-e223-11ea-86c5-bbf5e5b5c494.png)

**Comment by NailBash** [comment]
I noticed it. I didn't see it at first. Thank you for help, scheduler alive now.
P.S. Thats very strange, why log file can be created by not apache user?

**Comment by iglocska** [comment]
If you ever start the worker via the command line as root instead of going through MISP / the instructions detailed in the setup guide it will create the log file with the user that was used to run the worker.

**Comment by NailBash** [comment]
Well, now I can start scheduler, but it always dies when tryes to fetch feeds.

![изображение](https://user-images.githubusercontent.com/29750691/90847973-2fc7cc00-e385-11ea-9414-45fe3f7f00fa.png)

![изображение](https://user-images.githubusercontent.com/29750691/90848008-41a96f00-e385-11ea-8316-70fefd37d21b.png)

![изображение](https://user-images.githubusercontent.com/29750691/90848036-5128b800-e385-11ea-8822-6990fed8b766.png)

Launched by apache user
![изображение](https://user-images.githubusercontent.com/29750691/90848085-6867a580-e385-11ea-909e-d741c8566c7f.png)

**Comment by NailBash** [comment]
And after restart I get this error
![изображение](https://user-images.githubusercontent.com/29750691/90848265-e2982a00-e385-11ea-880d-fbc7c9a0c4db.png)

**Comment by NailBash** [comment]
I restarted rh-redis32-redis.service , rh-php72-php-fpm.service, httpd and misp-workers. After that I pushed "fetch feeds" button and feeds downloaded successfully (with the exception some of  them )
![изображение](https://user-images.githubusercontent.com/29750691/90849798-8e8f4480-e389-11ea-9bdb-acb363bc414c.png)

But this problem in feeds servers, I think. And I can't say if worker will work when time of feed fetching comes tomorrow.

P.S. Well, looks like scheduler works now.
